src/renderer/router/index.js

```diff
@@ -15,73 +31,7 @@ export default new Router({
             redirect: '/agent/:agentid/dids',
```

Comment:
This will bite us if `admin-dids` isn't supported by the connected agent; maybe we come up with a "general info about the agent" view to show by default? Removing the redirect and just letting the starting screen be blank might be good enough for now though.

Reply:
The blank looks odd, but in the Spirit of Not Perfect, I'm going to leave it blank and fix it as part of a new effort in the future.